packages/react-badge/src/components/Badge/Badge.types.ts

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { ComponentProps, ShorthandProps, ObjectShorthandProps } from '@fluentui/react-utilities';
+
+/**
+ * {@docCategory Badge}
+ */
+export type SizeValue = 'smallest' | 'smaller' | 'small' | 'medium' | 'large' | 'larger' | 'largest';
+
+/**
+ * {@docCategory Badge}
+ */
+export type BadgeStatus =
+  | 'success'
+  | 'warning'
+  | 'severe'
+  | 'accent'
+  | 'danger'
+  | 'important'
+  | 'informative'
+  | 'subtle';
+
+/**
+ * {@docCategory Badge}
+ */
+export type BadgeAppearances = 'filled' | 'outline' | 'ghost' | 'tint';
+
+/**
+ * {@docCategory Badge}
+ */
+export type BadgeShape = 'rounded' | 'square' | 'circular';
+
+/**
+ * {@docCategory Badge}
+ */
+export interface BadgeProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
+  /**
+   * A Badge can be sized.
+   */
```

Comment:
Include the default value in the documentation for these props. E.g. `@defaultvalue medium`